vm/src/thread.rs

```diff
@@ -753,10 +761,19 @@ impl<'b> OwnedContext<'b> {
                     let state = {
                         let mut context = context.borrow_mut();
 
+                        let instruction_index = context.stack.frame.instruction_index;
```

Comment:
Could assign `closure.function.required_stack_size` here too to reduce the size of the predicate at 768